### PR TITLE
Added client setttings header with actions

### PR DESCRIPTION
**Pull request by edewit**

### CODE REVIEW

File: src/clients/ClientSettings.tsx [hunk 1]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+              <TextInput
+                type="text"
+                id="kc-home-url"
+                name="baseUrl"
+                ref={form.register}
+              />
+            </FormGroup>
+          </Form>
+          <Form isHorizontal>
+            <FormGroup label={t("consentRequired")} fieldId="kc-consent">
```
Comment (srambach) [inline src/clients/ClientSettings.tsx:180]: Please add `hasNoPaddingTop` to the form group to align the switch

File: src/clients/ClientSettings.tsx [hunk 2]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+                    id="kc-consent"
+                    label={t("common:on")}
+                    labelOff={t("common:off")}
+                    isChecked={value}
+                    onChange={onChange}
+                  />
+                )}
+              />
+            </FormGroup>
+            <FormGroup
```
Comment (srambach) [inline src/clients/ClientSettings.tsx:196]: Please add `hasNoPaddingTop` to the form group to align the switch

File: src/clients/ClientSettings.tsx [hunk 3]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+                  } else if (value === "delete") {
+                    toggleDeleteDialog();
+                  }
+                }}
+              />
+            </>
+          );
+        }}
+      />
+      <PageSection>
```
Comment (srambach) [inline src/clients/ClientSettings.tsx:144]: I think this needs a `variant="light"`

File: src/clients/ClientSettings.tsx [hunk 4]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+            <>
+              <DisableConfirm />
+              <ViewHeader
+                titleKey="clients:clientList"
+                subKey="clients:clientsExplain"
+                selectItems={[
+                  <SelectOption
+                    key="action"
+                    value={t("common:action")}
+                    isPlaceholder={true}
```
Comment (ssilvert) [inline src/clients/ClientSettings.tsx:114]: `Action` should not show as a selectable value in the dropdown.

File: src/clients/ClientSettings.tsx [hunk 5]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+                    toggleDeleteDialog();
+                  }
+                }}
+              />
+            </>
+          );
+        }}
+      />
+      <PageSection>
+        <Alerts />
```
Comment (ssilvert) [inline src/clients/ClientSettings.tsx:145]: Not something to address in this PR, but I don't understand why we need to keep adding the Alerts component over and over.  I would think that we should have only one instance of the Alerts component that all screens could add alerts to.

**Review by ssilvert — CHANGES_REQUESTED** [review]
When you go to Client Settings page, the header still says `Client list`

File: src/clients/ClientSettings.tsx (hunk 4 above)
Comment (edewit) [reply src/clients/ClientSettings.tsx:114]: That is the way it is with patternfly even if you give add the `isPlaceholder` it add it to the dropdown. @srambach do you know how you can get a placeholder to not show up in the dropdown?

File: src/clients/ClientSettings.tsx (hunk 5 above)
Comment (edewit) [reply src/clients/ClientSettings.tsx:145]: I don't want to have a static component that we add like in the other console.

Comment (edewit) [reply src/clients/ClientSettings.tsx:145]: created issue #132 for this.

File: src/clients/ClientSettings.tsx (hunk 4 above)
Comment (edewit) [reply src/clients/ClientSettings.tsx]: ahhh... I see it's resolved in 37c1576

**Comment by ssilvert** [comment]
@srambach If you agree that all your issues are resolved then please approve.

File: src/clients/ClientSettings.tsx [hunk 6]
```diff
@@ -65,95 +85,152 @@ export const ClientSettings = () => {
...
+            onConfirm: () => {
+              onChange(!value);
+              save();
+            },
+          });
+          return (
+            <>
+              <DisableConfirm />
+              <ViewHeader
+                titleKey="clients:clientList"
```
Comment (srambach) [inline src/clients/ClientSettings.tsx:108]: Is this still the wrong page title?  
Is this the mockup, and it should be the client name and have breadcrumbs?

Comment (edewit) [reply src/clients/ClientSettings.tsx]: Changed to use the client name and the breadcrumbs are done in #119 and yes that is the mockup

Comment (srambach) [reply src/clients/ClientSettings.tsx]: I'm just seeing a blank now, even though there's a clientID shown in the list...?

Comment (edewit) [reply src/clients/ClientSettings.tsx]: strange it seems to works sometimes... now I really fixed it ;)

**Review by ssilvert — APPROVED** [review]

**Pull request merged**